tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/operative_species.tsx

```diff
@@ -2,5 +2,7 @@ import { CheckboxInput, FeatureToggle } from '../base';
 
 export const operative_species: FeatureToggle = {
   name: 'Always Human as Operative',
+  description:
+    'If true, you will always spawn as a human (this means your backup human if an alien species) when you roll nuclear operative.',
```

Comment:
Don't say "true", say "checked".

```suggestion
    'If checked, you will spawn as your backup human as nuclear operative.',
```

Reply:
i think the purpose of this is already implied via the name, so i'll cut it out unless you deem it useful

Reply:
Sounds good